solidity/contracts/converter/types/standard-pool/StandardPoolConverter.sol

```diff
@@ -230,7 +232,7 @@ contract StandardPoolConverter is
         // verify the the converter has exactly two reserves
         require(reserveTokenCount() == 2, "ERR_INVALID_RESERVE_COUNT");
         anchor.acceptOwnership();
-        syncReserveBalances();
+        syncReserveBalances(0);
```

Comment:
@lbeder : @yudilevi has opposed that in the past, so please convince him.

Reply:
yeah I don't like the implicit side effect of using msg.value in an internal function.
it can cause problems when the function is called from different flows.